Pylint: Update code to use f-strings where appropriate

## 💡 Description
Update the codebase to use f-strings where appropriate. `pylint` report is attached (`pylint . --disable=all --enable=consider-using-f-string`:
```
************* Module pds.naif_pds4_bundler.classes.collection.collection
pds/naif_pds4_bundler/classes/collection/collection.py:159:22: C0209: Formatting a regular string which could be an f-string (consider-using-f-string)
pds/naif_pds4_bundler/classes/collection/collection.py:177:22: C0209: Formatting a regular string which could be an f-string (consider-using-f-string)
pds/naif_pds4_bundler/classes/collection/collection.py:185:18: C0209: Formatting a regular string which could be an f-string (consider-using-f-string)
************* Module pds.naif_pds4_bundler.classes.label.pds4_metakernel
pds/naif_pds4_bundler/classes/label/pds4_metakernel.py:61:25: C0209: Formatting a regular string which could be an f-string (consider-using-f-string)
************* Module pds.naif_pds4_bundler.classes.label.pds3_inventory
pds/naif_pds4_bundler/classes/label/pds3_inventory.py:24:14: C0209: Formatting a regular string which could be an f-string (consider-using-f-string)
************* Module pds.naif_pds4_bundler.classes.label.label
pds/naif_pds4_bundler/classes/label/label.py:134:19: C0209: Formatting a regular string which could be an f-string (consider-using-f-string)
************* Module pds.naif_pds4_bundler.classes.product.product_metakernel
pds/naif_pds4_bundler/classes/product/product_metakernel.py:300:22: C0209: Formatting a regular string which could be an f-string (consider-using-f-string)
************* Module pds.naif_pds4_bundler.classes.product.product_inventory
pds/naif_pds4_bundler/classes/product/product_inventory.py:129:19: C0209: Formatting a regular string which could be an f-string (consider-using-f-string)
************* Module pds.naif_pds4_bundler.classes.product.product_orbnum
pds/naif_pds4_bundler/classes/product/product_orbnum.py:152:19: C0209: Formatting a regular string which could be an f-string (consider-using-f-string)
************* Module pds.naif_pds4_bundler.classes.product.product_kernel
pds/naif_pds4_bundler/classes/product/product_kernel.py:172:22: C0209: Formatting a regular string which could be an f-string (consider-using-f-string)
************* Module pds.naif_pds4_bundler.classes.product.product_checksum
pds/naif_pds4_bundler/classes/product/product_checksum.py:253:19: C0209: Formatting a regular string which could be an f-string (consider-using-f-string)
************* Module pds.naif_pds4_bundler.classes.product.product_spiceds
pds/naif_pds4_bundler/classes/product/product_spiceds.py:87:20: C0209: Formatting a regular string which could be an f-string (consider-using-f-string)
pds/naif_pds4_bundler/classes/product/product_spiceds.py:140:19: C0209: Formatting a regular string which could be an f-string (consider-using-f-string)
************* Module pds.naif_pds4_bundler.utils.time
pds/naif_pds4_bundler/utils/time.py:66:20: C0209: Formatting a regular string which could be an f-string (consider-using-f-string)
************* Module pds.naif_pds4_bundler.utils.files
pds/naif_pds4_bundler/utils/files.py:519:16: C0209: Formatting a regular string which could be an f-string (consider-using-f-string)
```

--- a/src/pds/naif_pds4_bundler/classes/collection/collection.py
+++ b/src/pds/naif_pds4_bundler/classes/collection/collection.py
@@ -157,7 +157,7 @@ class Collection:
                 else:
                     version = int(versions[-1].split("v")[-1].split(".")[0])
 
-                vid = "{}.0".format(version)
+                vid = f'{version}.0'
                 logging.info(
                     f"-- Collection of {self.type} version set to "
                     f"{version}, derived from:"
@@ -175,7 +175,7 @@ class Collection:
                     f"-- No {self.type} collection available in previous increment."
                 )
                 logging.warning(f"-- Collection of {self.type} version set to: {ver}.")
-                vid = "{}.0".format(ver)
+                vid = f'{ver}.0'
                 logging.info("")
 
         else:
@@ -183,7 +183,7 @@ class Collection:
                 f"-- Collection of {self.type} version set "
                 f"to: {int(self.setup.release)}."
             )
-            vid = "{}.0".format(int(self.setup.release))
+            vid = f'{int(self.setup.release)}.0'
             logging.info("")
 
         self.vid = vid

--- a/src/pds/naif_pds4_bundler/classes/label/label.py
+++ b/src/pds/naif_pds4_bundler/classes/label/label.py
@@ -132,7 +132,7 @@ class PDSLabel:
             #
             # Obtain all observers.
             #
-            obs = ["{}".format(self.setup.observer)]
+            obs = [f'{self.setup.observer}']
 
             if hasattr(self.setup, "secondary_observers"):
                 sec_obs = self.setup.secondary_observers

--- a/src/pds/naif_pds4_bundler/classes/label/pds3_inventory.py
+++ b/src/pds/naif_pds4_bundler/classes/label/pds3_inventory.py
@@ -19,10 +19,7 @@ class InventoryPDS3Label(PDSLabel):
 
         # TODO: Check why this template path is not following the approach of all
         #       other labels.
-        self.template = (
-            self.root_dir
-            + "/templates/pds3/template_collection_{}.lbl".format(collection.type)
-        )
+        self.template = f'{self.root_dir}/templates/pds3/template_collection_{collection.type}.lbl'
 
         self.VOLUME_ID = self.setup.volume_id
         self.ROW_BYTES = str(self.product.row_bytes)

--- a/src/pds/naif_pds4_bundler/classes/label/pds4_metakernel.py
+++ b/src/pds/naif_pds4_bundler/classes/label/pds4_metakernel.py
@@ -58,9 +58,7 @@ class MetaKernelPDS4Label(PDSLabel):
             # merely a list of strings.
             #
             kernel_type = extension_to_type(kernel)
-            kernel_lid = "{}:spice_kernels:{}_{}".format(
-                self.setup.logical_identifier, kernel_type, kernel.lower()
-            )
+            kernel_lid = f'{self.setup.logical_identifier}:spice_kernels:{kernel_type}_{kernel.lower()}'
 
             kernel_list_for_label += (
                 f"{' ' * 2 * tab}<Internal_Reference>{eol}"

--- a/src/pds/naif_pds4_bundler/classes/product/product_checksum.py
+++ b/src/pds/naif_pds4_bundler/classes/product/product_checksum.py
@@ -257,7 +257,7 @@ class ChecksumProduct(Product):
 
     def set_product_vid(self) -> None:
         """Set Product VID."""
-        self.vid = "{}.0".format(int(self.version))
+        self.vid = f'{int(self.version)}.0'
 
     def write_product(self, history: bool=False, set_coverage: bool=False) -> None:
         """Write the Checksum file and determine its start and stop time.

--- a/src/pds/naif_pds4_bundler/classes/product/product_inventory.py
+++ b/src/pds/naif_pds4_bundler/classes/product/product_inventory.py
@@ -132,7 +132,7 @@ class InventoryProduct(Product):
 
     def set_product_vid(self) -> None:
         """Set the Product VID."""
-        self.vid = "{}.0".format(int(self.version))
+        self.vid = f'{int(self.version)}.0'
 
     def write_product(self) -> None:
         """Write and validate the Collection inventory."""

--- a/src/pds/naif_pds4_bundler/classes/product/product_kernel.py
+++ b/src/pds/naif_pds4_bundler/classes/product/product_kernel.py
@@ -169,9 +169,7 @@ class SpiceKernelProduct(Product):
 
         :return: product LID
         """
-        product_lid = "{}:spice_kernels:{}_{}".format(
-            self.setup.logical_identifier, self.type, self.name
-        ).lower()
+        product_lid = f'{self.setup.logical_identifier}:spice_kernels:{self.type}_{self.name}'.lower()
 
         return product_lid
 

--- a/src/pds/naif_pds4_bundler/classes/product/product_metakernel.py
+++ b/src/pds/naif_pds4_bundler/classes/product/product_metakernel.py
@@ -297,9 +297,7 @@ class MetaKernelProduct(Product):
         else:
             name = self.name
 
-        product_lid = "{}:spice_kernels:{}_{}".format(
-            self.setup.logical_identifier, self.type, name
-        ).lower()
+        product_lid = f"{self.setup.logical_identifier}:spice_kernels:{self.type}_{name}".lower()
 
         self.lid = product_lid
 

--- a/src/pds/naif_pds4_bundler/classes/product/product_orbnum.py
+++ b/src/pds/naif_pds4_bundler/classes/product/product_orbnum.py
@@ -149,9 +149,7 @@ class OrbnumFileProduct(Product):
 
     def set_product_lid(self) -> None:
         """Set the Product LID."""
-        self.lid = "{}:miscellaneous:orbnum_{}".format(
-            self.setup.logical_identifier, self.name
-        ).lower()
+        self.lid = f'{self.setup.logical_identifier}:miscellaneous:orbnum_{self.name}'.lower()
 
     def set_product_vid(self) -> None:
         """Set the Product VID."""

--- a/src/pds/naif_pds4_bundler/classes/product/product_spiceds.py
+++ b/src/pds/naif_pds4_bundler/classes/product/product_spiceds.py
@@ -84,7 +84,7 @@ class SpicedsProduct(Product):
         # not remove a previously provided SPICEDS file from the configuration
         # file, if so, the user must be warned.
         #
-        self.name = "spiceds_v{0:0=3d}.html".format(self.version)
+        self.name = f'spiceds_v{self.version:0=3d}.html'
         self.path = (
             setup.staging_directory + os.sep + collection.name + os.sep + self.name
         )
@@ -137,7 +137,7 @@ class SpicedsProduct(Product):
 
     def set_product_vid(self) -> None:
         """Set the Product VID."""
-        self.vid = "{}.0".format(int(self.version))
+        self.vid = f'{int(self.version)}.0'
 
     def _check_cr(self) -> None:
         """Determine whether if ``<CR>`` has to be added to the SPICEDS."""

--- a/src/pds/naif_pds4_bundler/utils/files.py
+++ b/src/pds/naif_pds4_bundler/utils/files.py
@@ -515,9 +515,7 @@ def get_latest_kernel(
         try:
             return kernels.pop()
         except BaseException:
-            logging.warning(
-                f'        No kernels found with pattern {pattern}'
-            )
+            logging.warning("        No kernels found with pattern %s", pattern)
             return []
     else:
         #

--- a/src/pds/naif_pds4_bundler/utils/files.py
+++ b/src/pds/naif_pds4_bundler/utils/files.py
@@ -516,7 +516,7 @@ def get_latest_kernel(
             return kernels.pop()
         except BaseException:
             logging.warning(
-                "        No kernels found with pattern {}".format(pattern)
+                f'        No kernels found with pattern {pattern}'
             )
             return []
     else:

--- a/src/pds/naif_pds4_bundler/utils/time.py
+++ b/src/pds/naif_pds4_bundler/utils/time.py
@@ -63,7 +63,7 @@ def creation_time(time_format="infomod2"):
     """
     t = datetime.datetime.now()
     dt, micro = datetime.datetime.strftime(t, "%Y-%m-%dT%H:%M:%S.%f").split(".")
-    creation_time = "%s.%03d" % (dt, int(micro) / 1000) + "Z"
+    creation_time = f'{dt}.{int(micro) // 1000:03d}Z'
 
     if time_format == "maklabel":
         creation_time = creation_time[:-5]

